ND_Core/server/events.lua

```diff
@@ -1,5 +1,14 @@
 -- For support join my discord: https://discord.gg/Z9Mxu72zZ6
 
+-- Creating database tables
+AddEventHandler('onResourceStart', function(resourceName)
+    if (GetCurrentResourceName() ~= resourceName) then
+        return
+    end
+    MySQL.query("CREATE TABLE IF NOT EXISTS characters ( `character_id` INT(10) NOT NULL AUTO_INCREMENT, `license` VARCHAR(200) NOT NULL DEFAULT '0', `first_name` VARCHAR(50) NULL DEFAULT NULL, `last_name` VARCHAR(50) NULL DEFAULT NULL, `dob` VARCHAR(50) NULL DEFAULT NULL, `gender` VARCHAR(50) NULL DEFAULT NULL, `twt` VARCHAR(50) NULL DEFAULT NULL, `job` VARCHAR(50) NULL DEFAULT NULL, `cash` INT(10) NULL DEFAULT '0', `bank` INT(10) NULL DEFAULT '0', PRIMARY KEY (`character_id`) USING BTREE);")
+    print('^4ND_Core ^0Database structure validated!')
```

Comment:
What if there's an error, or what if they don't have oxmysql properly set up? This needs some error checking before validating.

Reply:
Yea, true. I can do some sort of thing